devicetypes/fibargroup/fibaro-dimmer-2-zw5.src/fibaro-dimmer-2-zw5.groovy

```diff
@@ -135,7 +135,11 @@ def ping(){
 }
 
 def installed(){
-  sendEvent(name: "checkInterval", value: 1920, displayed: false, data: [protocol: "zwave", hubHardwareId: device.hub.hardwareID])
+    sendEvent(name: "checkInterval", value: 1920, displayed: false, data: [protocol: "zwave", hubHardwareId: device.hub.hardwareID])
+}
+
+def configure(){
+	  sendEvent(name: "switch", value: "off", displayed: "true") //set the initial state to off.

```

Comment:
When possible, we try to get the actual state of the device here. I'd recommend something like

`response(refresh())` in the `installed()` method.

Reply:
@greens Thank you. I have made and test that change. The PR will be updated to include that.

Reply:
@greens This has been fixed and PR has been updated.